tests/explainers/test_kernel.py

```diff
@@ -32,17 +27,11 @@ def test_front_page_model_agnostic():
     explainer = shap.KernelExplainer(svm.predict_proba, X_train, nsamples=100, link="logit")
     shap_values = explainer.shap_values(X_test)
 
-    # plot the SHAP values for the Setosa output of the first instance
```

Comment:
👍 Was worried about the usefulness of this test, after this change... but saw there are other unmodified tests that just run the explainer, without asserting anything else; just like this